dashboard/src/main/webapp/src/pages/CoordinatorServerPage.vue

```diff
@@ -94,17 +95,27 @@ export default {
           serverInfo: {}
         }
     );
-    async function getCoordinatorServerConfPage() {
-      const res = await getCoordinatorConf();
+    async function getCoordinatorServerConfPage(headers) {
+      const res = await getCoordinatorConf({},headers)
       pageData.tableData = res.data.data
     }
-    async function getCoorServerInfo() {
-      const res = await getCoordinatorServerInfo();
+    async function getCoorServerInfo(headers) {
+      const res = await getCoordinatorServerInfo({},headers)
       pageData.serverInfo = res.data.data
     }
+
+    //The system obtains data from global variables and requests the interface to obtain new data after data changes.
+    const currentServerStore= useCurrentServerStore()
+    currentServerStore.$subscribe((mutable,state)=>{
+      const headrs={"targetAddress":state.currentServer}
```

Comment:
Is it better that passing this parameter through cookie？

Reply:
I think store `targetAddress` to `session` is better.

Reply:
> Is it better that passing this parameter through cookie？

This is global. The role of cookies is not well understood.

Reply:
It is not graceful to set the header before each request. Some code is redundant.

Reply:
Or maybe you can use local storage and get `targetAddress` from the local storage before invoking the http interfaces

Reply:
@yl09099 @zuston @rickyma  PTAL